azure-pipelines/main-pipeline.yml

```diff
@@ -62,6 +63,29 @@ stages:
     
     steps:
 
+    
+    # Static Analysis with SonarCloud. Pointing to Java Binaries and exluding the JDK
```

Comment:
spelling. also, this line and the next line look like two different sentences at first glance.
(not blocking)

Reply:
Fixed.